README.md

```diff
@@ -26,6 +26,12 @@ A project repo will be the place where all code, documentation and issue trackin
 
 - Bugfixes and new features should include tests[1].
 
+#### Migrations
+
+- Pull Requests should not contain migrations.  If schema changes are required, it should be indicated in the name of the PR or with a label.  Then, when the PR is reviewed and ready to merge, migrations should be added in a separate commit and merged immediately.
+
+- Migrations files should not be deleted or changed once they are in the master branch. 
```

Comment:
@skolsuper 
RE: what you suggested for migrations as a separate commit, I want to learn more about it. Please remind me when you see this message.

On the second point, we can `squashmigrations` if the app has not been launched and in production yet.
Please see:
https://docs.djangoproject.com/en/1.9/ref/django-admin/#django-admin-squashmigrations

CC: @elcolie